From 02bc80826f6363f9b0621a7e6c5ccd1f26c13f30 Mon Sep 17 00:00:00 2001
From: liuxiyao223 <liuxiyao223@huawei.com>
Date: Fri, 17 Nov 2023 17:42:21 +0800
Subject: [PATCH] Add http 3, and fix codecheck, P12, keyPasswd

Signed-off-by: liuxiyao223 <liuxiyao223@huawei.com>
---
 api/@ohos.net.http.d.ts | 31 ++++++++++++++++++++-----------
 1 file changed, 20 insertions(+), 11 deletions(-)

diff --git a/api/@ohos.net.http.d.ts b/api/@ohos.net.http.d.ts
index d52ab1af..bfe206ab 100644
--- a/api/@ohos.net.http.d.ts
+++ b/api/@ohos.net.http.d.ts
@@ -269,7 +269,7 @@ declare namespace http {
      * @crossplatform
      * @since 11
      */
-    PEM = "PEM",
+    PEM = 'PEM',
 
     /**
      * DER format certificate
@@ -277,15 +277,15 @@ declare namespace http {
      * @crossplatform
      * @since 11
      */
-    DER = "DER",
+    DER = 'DER',
 
     /**
-     * DER format certificate
+     * P12 format certificate
      * @syscap SystemCapability.Communication.NetStack
      * @crossplatform
      * @since 11
      */
-    P12 = "P12"
+    P12 = 'P12'
   }
 
   /**
@@ -297,12 +297,12 @@ declare namespace http {
    */
   export interface ClientCert {
     /**
-     * Cert: The absolute path to the client certificate file.
+     * The path to the client certificate file.
      * @type {string}
      * @syscap SystemCapability.Communication.NetStack
      * @since 11
      */
-    cert: string;
+    certPath: string;
 
     /**
      * The type of the client certificate.
@@ -313,15 +313,15 @@ declare namespace http {
     certType?: CertType;
 
     /**
-     * Key: The absolute path of the client certificate private key file.
-     * @type {string}
+     * The path of the client certificate private key file.
+     * @type {?string}
      * @syscap SystemCapability.Communication.NetStack
      * @since 11
      */
-    key: string;
+    keyPath?: string;
 
     /**
-     * KeyPasswd: Client certificate password.
+     * Password required to use the client certificate private key.
      * @type {?string}
      * @syscap SystemCapability.Communication.NetStack
      * @since 11
@@ -1516,7 +1516,16 @@ declare namespace http {
      * @crossplatform
      * @since 10
      */
-    HTTP2
+    HTTP2,
+
+    /**
+     * Protocol http3, for Use HTTP/3, fallback to HTTP/2 or HTTP/1 if needed.
+     * For HTTPS only. For HTTP, this option makes libcurl return error.
+     * @syscap SystemCapability.Communication.NetStack
+     * @crossplatform
+     * @since 11
+     */
+    HTTP3
   }
 
   /**
-- 
2.34.1

